scripts/build-package.ps1

```diff
@@ -33,11 +33,9 @@ function build_nodejs_client {
 
 # cd D:\a\aerospike-client-nodejs\aerospike-client-nodejs
  
-build_nodejs_client v10.20.0
-build_nodejs_client v12.22.10
-build_nodejs_client v14.19.0
-build_nodejs_client v16.14.0
-build_nodejs_client v17.8.0
-build_nodejs_client v18.0.0
+build_nodejs_client v14.21.1
+build_nodejs_client v16.18.1
+build_nodejs_client v18.12.1
+# build_nodejs_client v19.1.0
 
-nvm use v16.14.0
+nvm use v18.12.1
```

Comment:
I'm pretty sure we moved this entire flow to use github workflows a while back and not use the script anymore

Reply:
I will need to review the windows workflow, Ramaraj thought I should use this script, but it doesn't work as intended, and this comment may explain why.  I will review this workflow in a future iteration.

Reply:
By linux flow do you mean scripts/build-package.sh? This is used in Github workflows, but I also use this to create binaries for the NPM package.